### DISCUSSION TITLE
iOS Face/Person Search

### DISCUSSION THREAD

**Discussion by Menghini** [immich-app/immich#12573] [General]
Hello - Although I can search for a person by name in the web app, I cannot do so on the iOS mobile app. I can only see the list of people in the order of how many photos they have on my server. Being able to search for a person, is at least for me, one of the most common things I want to do when using the mobile app as I want to show pictures of who I am talking to. Furthermore, when I want to search for a person, I have to go to landscape mode on the web browser as the search field doesn't show up.

A long winded way to say that I am requesting that the mobile version of the app allow me to search for a person and to furthermore, make it easier to search in the web app on a small screen.

**Comment by alextran1502** [comment]
You can go in the `Search` page, click on the search input at the top, it will bring you to a more granular page where you can pick people/location/media type...etc

**Comment by Menghini** [comment]
I see that option when using the web version on my phone. I was unaware of that way to access it so thank you.

However, in my second image, I am using the iOS app and I cannot search for a person (at least, I haven't been able to figure that out) without scrolling to find the person I am looking for in the order of how many photos they have on my server. And to be more clear, in that image I am trying to search for myself and the people that show up doesn't change (as you can see, those other names clearly don't have "Travis" is the name). Am I missing something?
![image](https://github.com/user-attachments/assets/d864fabf-3997-4ab1-a474-932f56f7c8f4)
![image](https://github.com/user-attachments/assets/759cef33-4dd1-4749-aa57-402b47a42d95)

**Comment by Menghini** [comment]
Hello, I appreciate the new v1.118.0 update. However, I am still unable to search for people by typing their name when using the app. I find that because of this, I completely disregard the app and end up using the browser because it has better/more functionality. It would be nice to be able to search with more filters in the app by typing exactly what I want (right now the people are in order of how often they appear in photos).

**Comment by b-a0** [comment]
I have the same issue on Android. It would already be a large increase in convenience if the people were not sorted by number of tagged photos but simply by their name.
